.goreleaser.yaml

```diff
@@ -0,0 +1,38 @@
+project_name: fasttrackml
+builds:
+  - env:
+      - CGO_ENABLED=0
```

Comment:
I only started to look at your PR but I can see you disabled CGO here, which is required for FastTrackML to work — did you try launching any of the produced binaries?

Reply:
i just assumed that goreleaser did the needful by targeting os and arch -- but I'll copy the build settings from root Dockerfile